macros/configs/black_box_single.config.mac

```diff
@@ -0,0 +1,27 @@
+### -----------------------------------------------------------
```

Comment:
What's the purpose of this `configs` folder? Is this macro used? I don't see the correspondent `init` macro.

Reply:
This folder is just a residue of the first stages of the PR. I'll remove it since it has no purpose now.